### ISSUE TITLE
First xed admin:// requires password to be entered twice

### ISSUE THREAD

**Issue by ewangi** [linuxmint/xed#235]
```
 * Xed version 1.8.2
 * Distribution - Mint 19 MATE
```
When running xed admin:///path/to/file for the first time in a session, you are asked for your password twice. This is excessive. Any subsequent uses of admin:// in the session only require the password to be entered once.

**Steps to reproduce**

**Expected behaviour**

**Other information**

**Comment by icarter09** [comment]
Linux Mint 19.2
xed v2.2.3

I was able to reproduce this issue.

**Comment by icarter09** [comment]
Linux Mint 19,3
Xed v2.4.2

Just tested this out again and no longer able to reproduce the issue. @ewangi can you retest on the latest version of xed? Thanks.

**Comment by ewangi** [comment]
The problem still persists with me, @icarter09.

Linux Mint 19.3 MATE - up to date.
Xed 2.4.2

. Logout and login again.
. In terminal: xed admin:///etc/fstab
. Window "Authentication is required to perform file operations" opens - password entered.
. Above window re-opens - password entered a second time.
. File available for editing in xed.

**Comment by icarter09** [comment]
@ewangi you are right that it does occur with that /etc/fstab file. I also noticed that after it is asked twice, on the next attempt it is only asked for once.
